README.md

```diff
@@ -130,6 +130,7 @@ The above example uses app-level credentials i.e., publish profile file for depl
 Follow the steps to configure the secret:
 
 * Download the publish profile for the WebApp from the portal (Get Publish profile option)
+  * **Note**: As of October 2020, Linux web apps will need the app setting `WEBSITE_WEBDEPLOY_USE_SCM` set to `true`. This requirement will be removed in the future.
```

Comment:
The app setting change needs to happen BEFORE downloading the profile because that controls which endpoint they get.

Reply:
Updated the text